### DISCUSSION TITLE
Alternative WYSIWYG for Editor component (list indents)

### DISCUSSION THREAD

**Discussion by JoshuaDoshua** [primefaces/community#697] [PrimeVue] [answered]
Would love to be able to choose an alternative WYSIWYG. 

Quill doesn't handle list indents semantically (adds a padding-left indent, crazy custom css-counters, and a CSS dependency instead of properly nesting HTML, screenshot attached). I opened an [issue](https://github.com/quilljs/quill/issues/979) for this 7 years ago and this behavior still exists in the latest version.

I feel like I've seen others offer similar "pick your poison" for integrations, but WYSIWYGs aren't the easiest to work with. Is this something that would be up for discussion?

<img width="1366" alt="Screenshot 2023-12-14 at 5 22 03 PM" src="https://github.com/primefaces/community/assets/1930699/864f51ab-98e5-4ec2-92d0-6368b371c57f">
(from the playground on Quill's site)

**Comment by tugcekucukoglu** [comment, marked as answer]
We do not want to use 3rd party libraries in the future and we plan to write our own editor next year. Definitely on our roadmap.

Thanks,

**Reply by Remo** [reply]
@tugcekucukoglu is there an update regarding this? I'm looking for an alternative due to the same reason (strange semantics when building lists). 

I only found this: https://github.com/sfxcode/nuxt3-primevue-starter/blob/main/app/components/tiptap/TipTap.vue
In my experience, tiptap generates better code and might also be a good way to implement your own editor without spending a ton of time fixing the same problems others have fixed.

**Comment by JoseAntonioNavasSolutia** [comment]
when will it be available?
